docs/components/multiselect.md

```diff
@@ -24,6 +24,7 @@ An array of items containing a `label` and `value`.
 You may also specify:
 - a certain item is `selected` by default
 - a certain item is `disabled`
+- a certain item is readonly by using `isReadonly` property
```

Comment:
suggestion: can this be updated to `readonly` since the other properties do not use the `is*` prefix?

Reply:
I can update that, but since we have `attrs.readonly` as well `readonly class` used in the kongponent, I chose to keep `isReadonly` to avoid confusion. How does dismissable sound or should stick with `readonly` ?

Reply:
`attrs.readonly` shouldn't conflict since this is a property of an `item`, right?

I wouldn't go with `dismissable` since you're not really "dismissing" the item but removing it. 

You could do `removable`? 

---

I'm also wondering if you could instead just use the existing `disabled` property? 

- If `disabled: true` in the list, then you can't select it.
- If `disabled: true && selected: true` then you can't remove it.

Unsure if this logic makes sense though, but if yes, it would alleviate the need for another property.

Reply:
There is no conflict with attrs.readonly, just wanted to avoid any confusion. For the usecase that I am working with `disabled: true && selected: true` won't work because not every `disabled + selected item` needs to be `readonly`. `removable` sounds good.